### PR TITLE
Removed NIC stats collecting.

### PR DESCRIPTION
**Pull request by bjandras**

### CODE REVIEW

File: render/flowmill.render [hunk 1]
```diff
@@ -241,13 +241,7 @@ app agent_internal {
       5: u8 stream_type       // enum STREAM_TYPE { ST_SEND = 0, ST_RECV = 1 }
       6: u8 client_server     // enum CLIENT_SERVER_TYPE { SC_CLIENT = 0, SC_SERVER = 1 }
     }
-    25: log nic_queue_state {
```
Comment (jesafar) [inline render/flowmill.render:244]: We don't have to leave a 'gap' here to hold a position for older messages?  Or is the assumption that we're just pushing stuff 'afresh,' and so won't have older messages to deal with?

File: collector/kernel/bpf_src/render_bpf.c [hunk 2]
```diff
@@ -1738,93 +1672,6 @@ int on_tcp_reset(struct pt_regs *ctx, struct sock *sk)
   return 0;
 }
 
-int on_consume_skb(struct pt_regs *ctx, struct sk_buff *skb)
```
Comment (jesafar) [inline collector/kernel/bpf_src/render_bpf.c:1741]: Interesting.  We always return a '0' on error or not.  Is this a common pattern in the collector?

**Review by jesafar — APPROVED** [review]

File: render/flowmill.render (hunk 1 above)
Comment (bjandras) [reply render/flowmill.render:244]: These messages are internal to the kernel collector (sent from ebpf code to userspace), and are thus not part of some public interface that must be kept compatible with older versions.

File: collector/kernel/bpf_src/render_bpf.c (hunk 2 above)
Comment (bjandras) [reply collector/kernel/bpf_src/render_bpf.c:1741]: Not in the collector as a whole, just in the ebpf code for probe functions.

**Comment by jimwsplk** [comment]
LGTM

**Review by yonch — APPROVED** [review]
lgtm! thank you @bjandras !

**Pull request merged**